dataactbroker/handlers/fileHandler.py

```diff
@@ -785,6 +785,10 @@ def check_detached_generation(job_id):
             response_dict['status'] = 'invalid'
             response_dict['message'] = 'No generation job found with the specified ID'
             return JsonResponse.create(StatusCode.OK, response_dict)
+        elif upload_job.filename is None:
+            response_dict['status'] = 'invalid'
+            response_dict['message'] = 'No generation job found with the specified ID'
```

Comment:
If the filename is `NONE`, we should have a different message no? Since that doesn't mean `No generation job found with the specified ID`